lnd_test.go

```diff
@@ -13590,6 +13992,9 @@ var testsCases = []*testCase{
 		test: testExportChannelBackup,
 	},
 	{
+		name: "channel backup restore",
+		test: testChannelBackupRestore,
```

Comment:
we could make these test part of a new file `backup_test.go`, and assign a separate travis build for it.

Reply:
Yeah when we want to enable the remaining 24 test cases, that sounds like a good idea. But unless they run on some sort of "nightly" build, then they'll still add to the execution time of the integration tests.